### PR TITLE
aws-iam-authenticator 0.3.0

### PR DESCRIPTION
**Pull request by hakamadare**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

New package of aws-iam-authenticator.
-----

### CODE REVIEW

**Comment by hakamadare** [comment]
bump!

**Comment by Vlaaaaaaad** [comment]
Is there anything blocking this?

File: Formula/aws-iam-authenticator.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  def install
+    ENV["GOPATH"] = buildpath
+
+    (buildpath/"src/github.com/kubernetes-sigs/aws-iam-authenticator").install buildpath.children
+
+    # this works around an upstream bug
+    # already fixed, not yet released
+    # see https://github.com/kubernetes-sigs/aws-iam-authenticator/commit/678cdff0f43d3ae4a1d9d68a5347042f14a168a8
+    mkdir buildpath/"src/github.com/heptio"
+    ln_s buildpath/"src/github.com/kubernetes-sigs/aws-iam-authenticator", buildpath/"src/github.com/heptio/authenticator"
```
Comment (igas) [inline Formula/aws-iam-authenticator.rb]: @Moisan @SMillerDev should we use [install_symlink](https://docs.brew.sh/Formula-Cookbook#file-level-operations) here or it's not needed during build phase?

Comment (hakamadare) [reply Formula/aws-iam-authenticator.rb]: @igas if i understand correctly, `install_symlink` is for use when we want a symlink to exist in the final package.  in this case the symlink is temporary and will, i believe, be cleaned up after the build is finished.

**Comment by bdashrad** [comment]
It would be great to get this merged

**Comment by max-rocket-internet** [comment]
@lembacon any chance we could get this merged?

**Comment by themikenicholson** [comment]
Looking forward to this.

**Comment by SMillerDev** [comment]
@brewtestbot test this please!

**Comment by fxcoudert** [comment]
Thanks @hakamadare for the pull request

**Pull request closed without merging**